angular.json

```diff
@@ -317,6 +317,36 @@
           }
         }
       }
+    },
+    "clr-base": {
+      "projectType": "library",
+      "root": "src/clr-base",
+      "sourceRoot": "src/clr-base/src",
+      "prefix": "clrx",
```

Comment:
is `clrx` correct here 🤔 ?

Reply:
It's basically `tbd` at the moment. `clrx` is a placeholder until we land on a prefix in the near future.